app/src/editor/canvas/components/ModuleSidebar.jsx

```diff
@@ -102,8 +102,8 @@ export default withErrorBoundary(ErrorComponentView)(class ModuleSidebar extends
                                                         )) || (
                                                             /* Text */
                                                             <TextInput value={option.value} onChange={this.onChange(name)} disabled={!!isRunning}>
-                                                                {({ innerRef, ...props }) => (
-                                                                    <input id={id} type="text" {...props} ref={innerRef} />
```

Comment:
I didn't know what's the purpose of this `innerRef` here so I removed it. @timoxley, is that fine or did I break something?